gradle.properties

```diff
@@ -1,14 +1,26 @@
-org.gradle.jvmargs=-Xmx4g
+org.gradle.caching=true
+org.gradle.configureondemand=true
+org.gradle.jvmargs=-Xmx4g -Dfile.encoding=UTF-8
+org.gradle.parallel=true
 
 # AndroidX
 android.useAndroidX=true
 
+# Enable install debug apk manually
+android.injected.testOnly=false
+
 # Disable welcome message.
 systemProp.org.gradle.internal.launcher.welcomeMessageEnabled=false
 
 # Disable adding the standard library by default.
 kotlin.stdlib.default.dependency=false
 
+# Enhance speed of kotlin build
+kotlin.code.style=official
```

Comment:
How does enabling the official kotlin code style improve build speed?

Reply:
I was negligent, it should be separated